test/indexer/test_hca_indexer.py

```diff
@@ -246,12 +245,17 @@ def test_bundle_downgrade(self):
         self._assert_old_bundle(num_expected_new_contributions=5, ignore_aggregates=True)
         self._assert_new_bundle(num_expected_old_contributions=5)
 
-    def _assert_old_bundle(self, num_expected_new_contributions=0, ignore_aggregates=False, ignore_deletes=False):
+    def _assert_old_bundle(self,
+                           num_expected_new_contributions=0,
+                           num_expected_new_deleted_contributions=0,
+                           ignore_aggregates=False,
+                           ignore_deletes=False):
```

Comment:
I have the feeling this parameter is a) badly named (shouldn't it be called `expect_deletions`?) and b) made redundant by `num_expected_new_deleted_contributions`.

Reply:
You're right, it is redundant. I removed it.